Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -15,12 +18,45 @@ public Inventory()
 
         public Item[] GetItems(string name)
         {
-            return new Item[0];
+            var foundItems = new Item[1];
+            foreach (var item in _items)
+            {
+                if (item.GetName() == name)
+                {
+                    foundItems[0] = item;
+                }
+            }
+
+            return foundItems;
         }
```

Comment:
This should return all instances, not only the last one found. :)

Reply:
oke, here I have to think about because right now item[] does not have a method/function to add things and in my oponion there is also no logical place to place such a function.

Reply:
I think I found a solution for this.